developer/js/README.md

```diff
@@ -1,8 +1,11 @@
 Keyman Developer
 ================
 
-This contains the source code of Keyman development tools. This also
-contains `kmlmc`.
+This contains the source code of Keyman lexical model development tools:
+
+ - `kmlmc`
+ - `kmlmp`
+ - `kmlmi`
 
 In order to build [lexical models][], these tools must be built and
```

Comment:
`kmlmp` is used only by command line tools (`kmcomp` and `tike` also support compiling .model.kmp files). `kmlmi` is used only within the lexical-models repo. This perhaps should be noted here?